### DISCUSSION TITLE
I woud like to see a Form Submit Button Component

### DISCUSSION THREAD

**Discussion by StephanMeijer** [filamentphp/filament#2159] [Ideas]
I have the current issue that I am building a front-end for my application. In this front-end, the user should be able to search through submitted posts. I am building that using a Livewire component and the Form Schema, but currently I am missing a submit button component to easily submit my front-end form component.

**Comment by ryangjchandler** [comment]
@StephanMeijer You should just build your own button.

**Reply by StephanMeijer** [reply]
But why? Why isn't this just included in the package? It is an often use component in webapplications.

**Reply by ryangjchandler** [reply]
We don't provide a form tag or anything with the form builder. That's all down to the developer as well.

**Reply by StephanMeijer** [reply]
That's strange, as it is just another type of `<input>`, just like `text`, `checkbox`, `time`, etc.

**Reply by danharrin** [reply]
At the moment, we don't even have a button view in the forms package, and I've been avoiding adding one as it involves maintaining more duplicate code between our packages. However, soon we'll have one anyway because of #2057, so we might as well add a component for it
